test_integration/.gitignore

```diff
@@ -0,0 +1,44 @@
+# Miscellaneous
```

Comment:
I think much of the contents of this file may well be already in the root ignore file. Could you clean it up a bit? I tend to try to have sub-folder Git ignores only have the stuff they absolutely need and then keep the boilerplate ignores for the root.

Reply:
Good idea. I didn't even look at it. It's auto-generated by `flutter create ...`

Reply:
Done